### DISCUSSION TITLE
Set-PnPField usage for Taxonomy site column

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [pnp/powershell#2204] [Q&A]
I have about 1000 sites I need to set the only taxonomy site column to point to the right term set and set 'Allow 'Fill-in' choices' to No. 
I can't figure out how to do it. 
Some advises I got did not error but did nothing.

Any one knows how to do it.
I saw 'Add-PnPTaxonomyField commandlet' but no 'Set-PnPTaxonomyField'

Any idea is welcome

**Comment by matschundmodder** [comment]
Running into the same issue. Still seems impossible to update an existing taxonomy field? Have you managed to solve this issue in a clean way? Or does it still have to do the messy way via the schemaXML? :)

**Reply by Ofer-Gal** [reply]
No Same issue exists
